maven-extension/README.md

```diff
@@ -63,9 +63,10 @@ The Maven OpenTelemetry Extension supports a subset of the [OpenTelemetry auto c
 | otel.exporter.otlp.headers   | OTEL_EXPORTER_OTLP_HEADERS  | Key-value pairs separated by commas to pass as request headers on OTLP trace and metrics requests.        |
 | otel.exporter.otlp.timeout   | OTEL_EXPORTER_OTLP_TIMEOUT  | The maximum waiting time, in milliseconds, allowed to send each OTLP trace and metric batch. Default is `10000`.  |
 | otel.resource.attributes | OTEL_RESOURCE_ATTRIBUTES | Specify resource attributes in the following format: key1=val1,key2=val2,key3=val3 |
+| otel.maven.instrumentation.mojos.enabled | OTEL_MAVEN_INSTRUMENTATION_MOJOS_ENABLED | Whether to create spans for mojo goal executions, `true` or `false`. Can be configured to reduce the number of spans created for large builds. Default value: `true` |
```

Comment:
Based on the ones I see here: https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/config/common.md

I would agree it should probably be `OTEL_INSTRUMENTATION_MAVEN_MOJOS_ENABLED`

Reply:
Bit of a nitpick, but can it be `maven.mojos` instead of a hyphen separator?

If additional properties are added in the future it's good to have them all under `maven.*` and it wouldn't require a rename of this one to do that

Reply:
I understand the rationale of the `-` over `.` based on the linked document, but I wonder if it applies given there isn't a separate library of instrumentation for Maven vs Maven Mojo. I consider it to be a Maven instrumentation and not a Maven Mojo one, meaning `maven` is the instrumentation and not `maven-mojo`.

I could easily be splitting hairs on this, so please ignore me if that's the case.